cwl/src/main/scala/cwl/CommandOutputBinding.scala

```diff
@@ -63,9 +67,13 @@ case class CommandOutputBinding(
 
   private def load64KiB(path: String, ioFunctionSet: IoFunctionSet): String = {
     // This suggests the IoFunctionSet should have a length-limited read API as both CWL and WDL support this concept.
-    // val content = ioFunctionSet.readFile(path)
-    val content = better.files.File(path).bytes.take(64 * 1024).toArray
-    new String(content)
+    // ChrisL: But remember that they are different (WDL => failure, CWL => truncate)
```

Comment:
It's also a configurable length which needs to make its way from the Cromwell conf to here

Reply:
Not to here. This is a CWL spec thing and so hard-coding is fine (IMO)

Reply:
Ok yeah not for CWL but for WDL :)